frame/election-provider-support/benchmarking/src/lib.rs

```diff
@@ -88,4 +97,20 @@ benchmarks! {
 				::solve(d as usize, targets, voters).is_ok()
 		);
 	}
+
+	mms {
```

Comment:
Does this actually run in a reasonable about of time? if not, we should not benchmark this, and should not implement `Solver` for it either.